awd/cli.py

```diff
@@ -27,159 +27,81 @@
-    default=config.COLLECTION_HANDLE,
-    help="The handle of the DSpace collection to which items will be added.",
-)
 @click.pass_context
 def deposit(
     ctx: click.Context,
-    metadata_url: str,
-    content_url: str,
-    bucket: str,
-    sqs_input_queue: str,
-    collection_handle: str,
 ) -> None:
     """Process a text file of DOIs to retrieve metadata and PDFs and send to SQS queue.
 
     Errors generated during the process are emailed to stakeholders.
     """
-    date = datetime.datetime.now(tz=datetime.UTC).strftime(config.DATE_FORMAT)
+    date = datetime.datetime.now(tz=datetime.UTC).strftime(DATE_FORMAT)
     stream = ctx.obj["stream"]
+    config = ctx.obj["config"]
     s3_client = S3Client()
     sqs_client = SQSClient(
-        region=ctx.obj["aws_region"],
-        base_url=ctx.obj["sqs_base_url"],
-        queue_name=sqs_input_queue,
+        region=AWS_REGION_NAME,
+        base_url=config.SQS_BASE_URL,
+        queue_name=config.SQS_INPUT_QUEUE,
     )
-
     try:
-        s3_client.client.list_objects_v2(Bucket=bucket)
+        s3_client.client.list_objects_v2(Bucket=config.BUCKET)
     except ClientError as e:
         logger.exception(
-            "Error accessing bucket: %s, %s", bucket, e.response["Error"]["Message"]
+            "Error accessing bucket: %s, %s",
+            config.BUCKET,
+            e.response["Error"]["Message"],
         )
         return  # Unable to access S3 bucket, exit application
-
-    DoiProcessAttempt.set_table_name(ctx.obj["doi_table_name"])
+    DoiProcessAttempt.set_table_name(config.DOI_TABLE)
     if not DoiProcessAttempt.exists():
         logger.exception("Unable to read DynamoDB table")
         return  # exit application
 
-    for doi_file in s3_client.retrieve_file_type_from_bucket(bucket, ".csv", "archived"):
-        dois = get_dois_from_spreadsheet(f"s3://{bucket}/{doi_file}")
-
+    for doi_file in s3_client.retrieve_file_type_from_bucket(
+        config.BUCKET, ".csv", "archived"
+    ):
+        dois = get_dois_from_spreadsheet(f"s3://{config.BUCKET}/{doi_file}")
         for doi in dois:
+            logger.error(doi)
             article = Article(
                 doi=doi,
-                metadata_url=metadata_url,
-                content_url=content_url,
+                metadata_url=config.METADATA_URL,
+                content_url=config.CONTENT_URL,
                 s3_client=s3_client,
-                bucket=bucket,
+                bucket=config.BUCKET,
                 sqs_client=sqs_client,
-                sqs_base_url=ctx.obj["sqs_base_url"],
-                sqs_input_queue=sqs_input_queue,
-                sqs_output_queue=ctx.obj["sqs_output_queue"],
-                collection_handle=collection_handle,
+                sqs_base_url=config.SQS_BASE_URL,
+                sqs_input_queue=config.SQS_INPUT_QUEUE,
+                sqs_output_queue=config.SQS_OUTPUT_QUEUE,
+                collection_handle=config.COLLECTION_HANDLE,
```

Comment:
Just noting as I move through a bit... I like that the uppercase nature of these.  At a glance, this suggests to me that these are either constants or env vars, and it's clear they are some kind of baseline, config-y thing.